src/tools/tidy/src/extdeps.rs

```diff
@@ -4,7 +4,10 @@ use std::fs;
 use std::path::Path;
 
 /// List of allowed sources for packages.
-const ALLOWED_SOURCES: &[&str] = &["\"registry+https://github.com/rust-lang/crates.io-index\""];
+const ALLOWED_SOURCES: &[&str] = &[
+    "\"registry+https://github.com/rust-lang/crates.io-index\"",
+    "\"git+https://github.com/antoyo/gccjit.rs",
```

Comment:
That's not enough. We'd likely want to add that as a submodule and/or subtree; we don't want `git` sources here, only crates.io and in-tree copies. Otherwise we'd at minimum need to make sure our vendored tarballs are self-contained.

Reply:
@Mark-Simulacrum: I published a version of [gccjit.rs](https://crates.io/crates/gccjit) that can now be used by rustc_codegen_gcc.
Do we still need to move the repo to the rust-lang org if we use the crate on crates.io?

Reply:
Strictly speaking, no, but I'd generally prefer if we did (and made sure rust-lang-owner had publish access).

Reply:
(And t-compiler may have a stricter sense. From an infra perspective the important thing is that vendoring works and our dependencies are sourced in a way that won't stop working).

Reply:
> Strictly speaking, no, but I'd generally prefer if we did (and made sure rust-lang-owner had publish access).

You mean publish access to the repo or the crate gccjit.rs on crates.io?

Reply:
Publish access for the crates.io crate.

Reply:
Ok, I sent an invite to `rust-lang-owner` to have access to the crate on crates.io.